python/mxnet/ndarray/numpy/_op.py

```diff
@@ -2068,6 +2068,8 @@ def tril(m, k=0):
            [ 7.,  8.,  0.],
            [10., 11., 12.]])
     """
+    if isinstance(m, numeric_types):
+        raise TypeError("tril() missing 1 required positional argument: 'm'")
```

Comment:
I feel it might be better to have more specific error msg saying the it is expected to be at least 2D.
same for the remains.